announce/tests/test_bellman.py

```diff
@@ -241,15 +249,15 @@ def setUp(self):
 
     def test_announce(self):
         bm = Bellman(text='announce apple message text',
-                     user_name='bob',
-                     user_id='test_id')
+                     user_name='foo',
+                     user_id='1')
         group_name, space, text = bm.text.partition(' ')
 
         with patch.object(bm, 'send_announcement') as mock:
             bm.execute()
-            mock.assert_called()
+            mock.assert_called_with()
             self.assertEqual(bm.text, '\n'.join([
-                'Message from <@test_id> :',
-                '<@1> <@2> ',
+                'Message from <@1> :',
```

Comment:
Please add the group that's being announced to here as well?
I can be part of multiple groups and an announcement like "meeting postponed" without group context is bound to cause trouble.